.github/workflows/main.yml

```diff
@@ -1,41 +1,58 @@
 name: CI
+
 on:
   pull_request:
     branches:
     - master
   push:
     branches:
     - master
+  schedule:
+    - cron: '0 10 * * *' # run at 10 AM UTC every day.
```

Comment:
So this runs tests against the latest master every day. What happens when it fails? Where does that report go?

Doing this might be a nice stop-gap, but the real test will be to run the `-models` tests for each PR in the core library.

Reply:
Yea, we might be able to trigger this build from pushes in the core through the GitHub API. Just speculating, I haven't looked into it yet.